### PR TITLE
History

### PR DESCRIPTION
**Pull request by keachi**
Adding the history feature. This will make it possible to store events like store and view when new customers or new servers were created or when the quota was changed.

### CODE REVIEW

File: bkpmgmt/bkpmgmt.py [hunk 1]
```diff
@@ -2,9 +2,21 @@
...
 import argparse
+import configparser
 import logging
+import os
 import sys
 
+from bkpmgmt import history
+
+CONFIG_USER = '~/.config/adfinis-bkpmgmt.ini'
+CONFIG_SYSTEM = '/etc/adfinis-bkpmgmt.ini'
```
Comment (winpat) [inline bkpmgmt/bkpmgmt.py]: Are you sure that we need the `adfinis-` Prefix?

File: bkpmgmt/bkpmgmt.py [hunk 2]
```diff
@@ -2,9 +2,21 @@
...
+
+CONFIG = configparser.ConfigParser()
+CONFIG['database'] = {
+    'path': '/var/lib/adfinis-bkpmgmt.db',
```
Comment (winpat) [inline bkpmgmt/bkpmgmt.py]: See above.

File: bkpmgmt/history.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        :param string vault:    Vault name or server hostname.
+        :param string size:     Quota size.
+
+        :returns: True
+        :rtype: bool
+
+        :raises sqlite3.OperationalError: if something with the database goes
+        wrong.
+        """
+        self._conn = sqlite3.connect(self._path)
```
Comment (winpat) [inline bkpmgmt/history.py]: Would it not be simpler to open up the connection in the constructor and leave it open?

**Comment by keachi** [comment]
@winpat I implemented your change requests.

**Pull request merged**